.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Thanks! I used to write how you did/do, but I found it really problematic when I came back to an old project and had a hard time reading thru what I did